Jenkinsfile

```diff
@@ -69,7 +69,8 @@ node {
           string(credentialsId: 'payment-queue-access-key-id-listen', variable: 'paymentQueueAccessKeyId'),
```

Comment:
for the deployment you no longer need the access keys or IDs passing setting as they are no longer needed.

Rather than passing the full queue url, the queue endpoint and the queue name can be passed for AWS, as the queue URL will be built up by concatenating the two. Here is all that is being passed to the node version:

```
  withCredentials([
    string(credentialsId: 'sqs-queue-endpoint', variable: 'sqsQueueEndpoint'),
    string(credentialsId: 'schedule-queue-name-pr', variable: 'scheduleQueueName'),
    string(credentialsId: 'payment-queue-name-pr', variable: 'paymentQueueName'),
    string(credentialsId: 'postgres-external-name-pr', variable: 'postgresExternalName'),
    string(credentialsId: 'payment-service-account-role-arn', variable: 'serviceAccountRoleArn'),
    usernamePassword(credentialsId: 'payment-service-postgres-user-pr', usernameVariable: 'postgresUsername', passwordVariable: 'postgresPassword'),
  ]) {
```

Reply:
Removed

Reply:
you're missing setting the `schedule-queue-name-pr` and `payment-queue-name-pr` in the jenkins file so the queue name is being left as 'payment' and 'schedule' in the PR deployment.

Reply:
Thanks good spot, I've updated added them.